platforms/ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -131,10 +131,10 @@ - (void)clearDataSource:(std::shared_ptr<Tangram::TileSource>)tileSource
 
 - (Tangram::SceneReadyCallback) sceneReadyListener {
     return [=](int sceneID, auto sceneError) {
-        if (!self.mapViewDelegate || ![self.mapViewDelegate respondsToSelector:@selector(mapView:didLoadScene:withError:)]) { return; }
-
         [self.markersById removeAllObjects];
```

Comment:
Good catch, we would benefit from leaving`self.renderRequested = YES;` before checking whether a `mapViewDelegate` is set.